team/karthik-venkataramani.yaml

```diff
@@ -0,0 +1,19 @@
+- title: Karthik Venkataramani
+  avatar: https://avatars.githubusercontent.com/kvenkman
+  role: Postdoctoral Fellow
+  organizations:
+    - name: University of Washington
+  bio: >
+    My work focuses on developing machine learning models for use in geospatial applications.
+  expertise:
+    - Remote Sensing
+    - Machine Learning
+    - Geospatial Data analysis
+  social:
+    - icon: github
+      link: https://github.com/kvenkman
+    - icon: linkedin
+      link: https://www.linkedin.com/in/karthikvenkataramani/
+  user_groups:
+    # From https://escience.washington.edu/using-data-science/hackweeks/planning/hackweek-roles
+    - Helper
```

Comment:
@scottyhq - Looks like we have an issue with the YAML here.
Should look like this:
```
  roles:
    - Helper
```

Reply:
I don't think that's it

https://github.com/uwhackweek/event-page-2024/blob/099ce079e3ce59603b8e921567796408e3be561d/team/template.yaml#L20-L23

maybe some special character situation or subtle thing with the way individual files get merged...

Reply:
> I don't think that's it
> 
> https://github.com/uwhackweek/event-page-2024/blob/099ce079e3ce59603b8e921567796408e3be561d/team/template.yaml#L20-L23
> 
> maybe some special character situation or subtle thing with the way individual files get merged...

Oh .. yeah. That can't be it.
Let me have another look